zk-tests/src/MockedModifier.t.sol

```diff
@@ -0,0 +1,100 @@
+// SPDX-License-Identifier: UNLICENSED
+
+pragma solidity ^0.8.13;
+import {Test, console2 as console} from "forge-std/Test.sol";
+
+contract MockInner {
+    // failure happens when returning >=5 items
```

Comment:
nit: not needed anymore?

Reply:
Yeah, and the test case should remain in codebase and any future reader can understand why the modifier signature has 5 items. Otherwise they might wanna simplify it.

Reply:
Maybe we should reword it that "potential failure happens..." or something similar